[DOCS-12704] update step duration documentation (#33171)
DataDog/documentation: content/en/synthetics/browser_tests/test_results.md

### Step duration

> The step duration represents the amount of time the step takes to be considered fully loaded, using the [Datadog locator system][9]. For more information, see [How Step Duration is Determined in Browser Tests][25].

Reviewer: ```suggestion
Step duration represents the time a step takes to be considered fully loaded using the [Datadog locator system][9]. For more information, see [How Step Duration is Determined in Browser Tests][25].
```
